web/src/message_fetch.js

```diff
@@ -83,7 +83,7 @@ function process_result(data, opts) {
     if (
         message_lists.current !== undefined &&
         opts.msg_list === message_lists.current &&
-        opts.msg_list.narrowed &&
+        opts.msg_list.not_combined_feed_view &&
```

Comment:
I think it's still better to spells this `!msg_list.is_combined_feed_view`.

Reply:
Okay, updated!